src/color/webglColormapGenerator.js

```diff
@@ -61,23 +61,22 @@ module.exports = function generateWebglColormapFunction(colormapName, colormapOb
 
         // TODO: scale from 0-1 to 0-255 with 0.0 precision
         var fn = '\
-vec4 ' + colormapName + '(float n) {\
-  vec2 x = vec2('+inMin+', '+inMax+');                    // input value min and max\
-  vec3 y0 = vec3('+outMinR+', '+outMinG+', '+outMinB+') / 255.0; // min output val\
-  vec3 y1 = vec3('+outMaxR+', '+outMaxG+', '+outMaxB+') / 255.0; // max output val'
+        vec4 ' + colormapName + '(float n) {\
+        vec2 x = vec2('+inMin+', '+inMax+');                    // input value min and max\
+        vec3 y0 = vec3('+outMinR+', '+outMinG+', '+outMinB+') / 255.0; // min output val\
+        vec3 y1 = vec3('+outMaxR+', '+outMaxG+', '+outMaxB+') / 255.0; // max output val'
 
-      } else {
+      } 
+      else {
 
         fn += '\
-  if ((n >= ' + inMin + ') && (n < ' + inMax + ')) {             // input value min and max\
-    x = vec2('+inMin+', '+inMax+');                    // input value min and max in vec2\
-    y0 = vec3('+outMinR+', '+outMinG+', '+outMinB+') / 255.0; // min output val\
-    y1 = vec3('+outMaxR+', '+outMaxG+', '+outMaxB+') / 255.0; // max output val'
-  }';
-
-      }
-    });
-  }
+        if ((n >= ' + inMin + ') && (n < ' + inMax + ')) {             // input value min and max\
+          x = vec2('+inMin+', '+inMax+');                    // input value min and max in vec2\
+          y0 = vec3('+outMinR+', '+outMinG+', '+outMinB+') / 255.0; // min output val\
+          y1 = vec3('+outMaxR+', '+outMaxG+', '+outMaxB+') / 255.0; // max output val'
+      };
```

Comment:
One last thing @Deepthi562. Line 77 should be like this:
```
}';
```
And please remove the single quote behind `val` on line 76. Thank you!!